lib/pronto/formatter/github_pull_request_review_formatter.rb

```diff
@@ -19,9 +19,8 @@ def submit_comments(client, comments)
         $stderr.puts "Failed to post: #{e.message}"
       end
 
-      def line_number(message, patches)
-        line = patches.find_line(message.full_path, message.line.new_lineno)
-        line.position
+      def line_number(message, _)
+        message.line.new_lineno if message.line
```

Comment:
Style/SafeNavigation: Use safe navigation (`&.`) instead of checking if an object exists before calling the method.